e2e-tests/functions

```diff
@@ -12,6 +12,7 @@ IMAGE_MONGOD=${IMAGE_MONGOD:-"perconalab/percona-server-mongodb-operator:main-mo
 IMAGE_BACKUP=${IMAGE_BACKUP:-"perconalab/percona-server-mongodb-operator:main-backup"}
 SKIP_BACKUPS_TO_AWS_GCP_AZURE=${SKIP_BACKUPS_TO_AWS_GCP_AZURE:-1}
 PMM_SERVER_VER="2.24.0"
+ENABLE_LOGGING="true"
```

Comment:
please enable it from `Jenkinsfile`, in most cases it is not needed when you run test manually